rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :rental, :date, :person, :book
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] When a rental is created it should add itself to the `rentals` arrays of the book and person objects. Kindly implement that.

  ```
  def initialize(date, book, person)
    @date = date
    @book = book
    @person = person
    person.rentals << self
    book.rentals << self
  end